backend/api.test/EventHandlers/TestMissionEventHandler.cs

```diff
@@ -414,15 +412,17 @@ public async Task QueuedMissionsAreNotAbortedWhenRobotAvailableHappensAtTheSameT
             Assert.Equal(MissionStatus.Pending, postTestMissionRun2!.Status);
         }
 
-        [Fact]
+#pragma warning disable xUnit1004
+        [Fact(Skip = "Skipping until issue #1767 is solved because test is unreliable")]
+#pragma warning restore xUnit1004
```

Comment:
We should make an issue on this before merging

Reply:
Created issue

https://github.com/equinor/flotilla/issues/1770